resolwe_bio/tools/append_sample_info.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+"""Extract reference (FASTA) and sample names from the VCF file."""
+
+import os
+import argparse
+
+from pysam import VariantFile
+from resolwe_runtime_utils import error, warning
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('vcf_file', help="VCF file (can be compressed using gzip/bgzip).")
+parser.add_argument('summary', help="Summary file to append to.")
+args = parser.parse_args()
+
+try:
+    vcf = VariantFile(args.vcf_file)
+except (OSError, ValueError):
+    print(error('Input VCF file does not exist or could not be correctly opened.'))
+    exit(1)
```

Comment:
Would this work in place of `exit(1)`?

```
error_msg = 'Input VCF file does not exist or could not be correctly opened.'
print(error_msg)
raise ValueError(error_msg)
```

Reply:
I have refactored so that both the process error (set manually) and error msg from the parser itself are presented to the user.